src/Appwrite/Event/Event.php

```diff
@@ -286,13 +286,6 @@ public function setParam(string $key, mixed $value): self
         return $this;
     }
 
-    public function setParamSensitive(string $key): self
```

Comment:
why did we remove this 👀

Reply:
It's unsed, we use this method with the extra param instead: https://github.com/appwrite/appwrite/blob/1.6.x/src/Appwrite/Event/Event.php#L206-L215